blackbox/docs/general/dql/joins.rst

```diff
@@ -201,17 +201,48 @@ Example with ``within`` scalar function::
     +--------------+---------+
     SELECT 2 rows in set (... sec)
 
-However, there are no specific optimisations for certain join conditions such
-as ``=`` (equals) yet. The JOIN operation is implemented as a generic nested
-loop that invokes the operation on every record of the left table with every
-record of the right table. However, these filters are applied in a distributed
-way for each shard of the used tables.
-
 Limitations
 ...........
 
  - Joining more than 2 tables can result in execution plans which perform
    poorly as there is no query optimizer in place yet.
 
+Available Join Algorithms
+-------------------------
+
+Nested-Loop Join Algorithm
+..........................
+
+The nested loop algorithm evaluates the join conditions on every record of the
+left table with every record of the right table in a distributed manner (for 
+each shard of the used tables). The right table is scanned once for every row
+in the left table.
```

Comment:
please mention that this is always the default algorithm if no one else can be executed